src/RobotPool.cpp

```diff
@@ -143,23 +143,22 @@ bool RobotPool::robotAvailable(RobotType _type)
 	switch (_type)
 	{
 	case ROBOT_DIGGER:
+	{
 		return getDigger() != nullptr;
-		break;
-
+	}
```

Comment:
I believe the `break` is still needed here.

Reply:
The return statement makes the `break;` unreachable.

Reply:
Oh geeze, right.